src/Workspaces/CSharpTest/CodeGeneration/AddImportsTests.cs

```diff
@@ -55,6 +54,9 @@ private async Task<Document> GetDocument(string code, bool withAnnotations)
 
         private async Task TestAsync(string initialText, string importsAddedText, string simplifiedText, bool safe, bool useSymbolAnnotations, Func<OptionSet, OptionSet> optionsTransform = null)
         {
+            if (safe != useSymbolAnnotations)
+                return;
```

Comment:
this was the quick and dirty way of ensuring these values only match what the product does without updating every test in this commit.  i'll update all the tests later on in the PR.

Reply:
Does this need to be removed now?

Reply:
has been removed.